### ISSUE TITLE
Produce coverage report if environment variable COVERAGE is set to true before running `ember test`

### ISSUE THREAD

**Issue by anubhawshrivastav** [kategengler/ember-cli-code-coverage#104]
If I set environment variable COVERAGE=true, I should be able to run `ember test` and get code-coverage output. That is not happening.

These are the steps to reproduce this issue:
`COVERAGE=true`
`echo $COVERAGE` 
> outputs `true` 

`ember test`
> does not produce code coverage report

**Comment by rwjblue** [comment]
Do you have ember-cli-code-coverage listed as a dependency or devDependency of your project?

Can you create a demo repo that we can use to help triage?  Anecdotally, this is working (I just tested on a fresh project)...

**Comment by anubhawshrivastav** [comment]
Interesting! I'm trying to make this run on Jenkins. I set COVERAGE env variable to true, and then in build script, try to run `ember test`. I do have `ember-cli-code-coverage` in `devDependencies`.

Just to be clear, with COVERAGE env var set to true, you were able to get code-coverage report by running `ember test` command (not specifying COVERAGE=true in it), right?

**Comment by anubhawshrivastav** [comment]
tl;dr RESOLVED!

There were issues with the Jenkins build environment, it was using manta, and so env var had to be set in a "special" way. I was able to do it, and it is now working on Jenkins. Thanks!!
